src/elements/Accordion/accordion.less

```diff
@@ -0,0 +1,55 @@
+/**
+ * Accordion
+ */
+
+@ui-accordion-background: @ui-color-turquoise;
+@ui-accordion-notification-border: 2px solid @ui-color-turquoise-dark;
+@ui-accordion-border: 2px solid @ui-color-gray-light;
+@ui-accordion-toggle-color: @ui-color-blue;
+
+.ui-accordion {
+	border-top:  @ui-accordion-border;
+	padding: @ui-small @ui-small 0;
+	margin: 0 -@ui-small;
+
+	&.notification {
+		background: @ui-accordion-background;
+		border: @ui-accordion-notification-border;
+		margin: 0;
+		padding: @ui-small;
+
+		.title {
+			color: @color-black;
+		}
+	}
+
+	.title {
+		font-weight: bold;
```

Comment:
Let's use the numeric value for the font weights (700 for bold). I'll probably make this a linting rule.